ethpm_cli/commands/registry.py

```diff
@@ -203,3 +216,15 @@ def generate_registry_store_data(
         "alias": alias,
         "active": activate,
     }
+
+
+def display_packages(all_package_names: Iterable[str], config: Config) -> None:
```

Comment:
You could probably get away with just passing the `w3` that exists on `config` here rather than the entire `config` instance - just to tidy things up a bit.

Reply:
Good call